server/routes/visit.ts

```diff
@@ -22,7 +21,7 @@ import MainContact from './visitJourney/mainContact'
 import RequestMethod from './visitJourney/requestMethod'
 import sessionCheckMiddleware from '../middleware/sessionCheckMiddleware'
 import type { Services } from '../services'
-import { eventAuditTypesOriginal, eventAuditTypesWithReview } from '../constants/eventAuditTypes'
+import { eventAuditTypesWithReview } from '../constants/eventAuditTypes'
```

Comment:
Looks like you could rename this now to just `eventAuditTypes` and tidy up in `server/constants/eventAuditTypes.ts` to get rid of the two different sets of values.